CHANGELOG.rst

```diff
@@ -1,3 +1,3 @@
-**0.1.0 - 05/01/24**
```

Comment:
My intent here is to start fresh and just re-use v0.1.0 tag to actually publish to pypi as v0.1.0. In that way I don't need to sort out all of the changes made since May. 

I'm happy to release v0.2.0 w/ actual changes documented since may if folks prefer though!